src/pages/home/report/ContextMenu/BaseReportActionContextMenu.tsx

```diff
@@ -144,7 +145,16 @@ function BaseReportActionContextMenu({
 
     const childReport = ReportUtils.getReport(reportAction?.childReportID ?? '-1');
     const parentReportAction = ReportActionsUtils.getReportAction(childReport?.parentReportID ?? '', childReport?.parentReportActionID ?? '');
-    const {reportActions: paginatedReportActions} = usePaginatedReportActions(childReport?.reportID ?? '-1');
+    const {reportActions: paginatedReportActions, sortedAllReportActions} = usePaginatedReportActions(childReport?.reportID ?? '-1');
+    const hasNewestReportAction = sortedAllReportActions && sortedAllReportActions[0]?.created === childReport?.lastVisibleActionCreated;
+
+    useEffect(() => {
+        if (hasNewestReportAction ?? !childReport?.reportID) {
+            return;
+        }
+
+        Report.openReport(childReport.reportID);
+    }, [hasNewestReportAction, childReport?.reportID]);
```

Comment:
I do agree, I also wasn’t sure about using `openReport` here to avoid unnecessary API calls, but with what we currently have implemented I didn’t find any other solution.

Could you create a thread for a discussion about it on slack or should I do it?